src/lib/swapper/swappers/ThorchainSwapper/getThorTradeQuote/getTradeQuote.ts

```diff
@@ -50,25 +48,18 @@ type ThorTradeQuote =
 
 export const getThorTradeQuote = async (
   input: GetTradeQuoteInput & { wallet?: HDWallet },
-  rates: Rates,
 ): Promise<Result<ThorTradeQuote, SwapErrorRight>> => {
   const {
     sellAsset,
     buyAsset,
     sellAmountIncludingProtocolFeesCryptoBaseUnit: sellAmountCryptoBaseUnit,
-    slippageTolerancePercentage,
     accountNumber,
     chainId,
     receiveAddress,
     affiliateBps,
     wallet,
   } = input
 
-  const slippageTolerance =
```

Comment:
TODO: put me in the quote request as `tolerance_bps`.

Reply:
Nope, don't do this `tolerance_bps` isn't actually slippage. Calculate it manually (https://github.com/shapeshift/web/pull/5094/commits/c84faf2f1b2d974a1228dc33c41abea217ad08e9).